static/js/runtime-config.js

```diff
@@ -32,15 +32,19 @@ export default class RuntimeConfig {
   set (key, value) {
     this._validateSet(key, value);
     this._data[key] = value;
-    this._onUpdateListener(this.getAll());
+    if (!this._onUpdateListeners[key]) {
+      this._onUpdate(key, () => {});
+    }
+    this._onUpdateListeners[key](value);
```

Comment:
Is there a reason for setting the listener if it doesn't exist rather than having the logic call the listener only if it exists?

Reply:
ah yea, good point..I will update that!